### DISCUSSION TITLE
Same material applied to different objects not working in Scene Viewer

### DISCUSSION THREAD

**Discussion by areavisuale** [google/model-viewer#3654] [Q&A]
### Description

I noticed that glb/gltf files that have 2 or more objects with the same material applied are not working in sceneviewer. (I remember that it worked before... I exported the 3d model from cinema4D) If I load these files in https://arvr.google.com/scene-viewer-preview nothing is shown, but there aren't errors. When I load these files in a android device I can see the model viewer part, but when I try to go in AR I get a message of error.

#### Live Demo

https://www.areavisuale.it/ar/webxr.html

the model in question is: https://www.areavisuale.it/ar/assets/cube2.glb

### Browser Affected

- [x] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] Helios
- [ ] IE
- [ ] Safari

### OS

- [x] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### Versions

- model-viewer: v0.10.0
- three.js: rXX

**Comment by areavisuale** [comment]
also models that have only a color instead of a texture in the color channel are not working in scene viewer... this is strange...

**Comment by elalish** [comment]
@areavisuale Interesting that nothing shows up in the scene-viewer-preview. @tpsiaki can you give any details here? Maybe a glTF parsing error?
